source/wp-content/themes/wporg-developer-2023/src/command-content/block.php

```diff
@@ -0,0 +1,108 @@
+<?php
+namespace WordPressdotorg\Theme\Developer_2023\Dynamic_Command_Content;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registers the block using the metadata loaded from the `block.json` file.
+ * Behind the scenes, it registers also all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/reference/functions/register_block_type/
+ */
+function init() {
+	register_block_type(
+		dirname( dirname( __DIR__ ) ) . '/build/command-content',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render',
+		)
+	);
+}
+
+/**
+ * Render the block content.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+
+	$content = filter_the_content( get_the_content() );
+
+	$wrapper_attributes = get_block_wrapper_attributes();
+	$repo_url = get_post_meta( get_the_ID(), 'repo_url', true );
+
+	if ( is_bundled_commands( $repo_url ) ) {
+		return sprintf(
+			'<section %1$s>%2$s</section>',
+			$wrapper_attributes,
+			do_blocks( $content ),
+		);
+	}
+
+	$repo_slug = str_replace( 'https://github.com/', '', $repo_url );
+	$command = get_the_title();
+	$installing_instructions = sprintf(
+		'
+		<!-- wp:heading {"level":3} --><h3 class="wp-block-heading">%1$s</h3><!-- /wp:heading -->
```

Comment:
Let's make all the headings match the single template. [Example](https://github.com/WordPress/wporg-developer/blob/2ce6c3f0fa837d86e41c60544511051748d9a9fa/source/wp-content/themes/wporg-developer-2023/src/code-comments/block.php#L52)

Reply:
Not sure if the changes [in this commit](https://github.com/WordPress/wporg-developer/pull/211/commits/9191e5d9f574ba126820b66d526eac91605acb66) match what you were suggesting. Correct me if I misunderstood.

Reply:
Which commit? That link isn't working.